projects/robots/telerob/telemax_pro/controllers/telemax_demo/telemax_demo.c

```diff
@@ -24,9 +24,10 @@
 
 #define TIME_STEP 32
 
-void wait(int time) {
+void noop(int n_steps) {
```

Comment:
Sorry to come late, but this variable seems poorly named and misleading. I would rename it to `duration` instead.